source/connect/connection-options.txt

```diff
@@ -42,426 +50,1319 @@ connection options in the string as ``<name>=<value>`` pairs. In the following e
+
+   .. tab:: MongoUrlBuilder
+      :tabid: mongo-url-builder
+
+      MaxConnecting
+      ~~~~~~~~~~~~~
+
+      .. include:: /includes/fundamentals/connection-options/maxConnecting.rst
+
+      .. literalinclude:: /includes/fundamentals/code-examples/connection/ConnectionOptions.cs
+         :language: csharp
+         :copyable: true
+         :dedent: 8
+         :start-after: // start-builder-max-connecting
+         :end-before: // end-builder-max-connecting
+
+      MaxConnectionIdleTime
+      ~~~~~~~~~~~~~~~~~~~~~
+
+      .. include:: /includes/fundamentals/connection-options/maxConnectionIdleTime.rst
+
+      .. literalinclude:: /includes/fundamentals/code-examples/connection/ConnectionOptions.cs
+         :language: csharp
+         :copyable: true
+         :dedent: 8
+         :start-after: // start-builder-max-connection-idle-time
+         :end-before: // end-builder-max-connection-idle-time
+
+      MaxConnectionLifeTime
+      ~~~~~~~~~~~~~~~~~~~~~
+
+      .. include:: /includes/fundamentals/connection-options/maxConnectionLifetime.rst
+
+      .. literalinclude:: /includes/fundamentals/code-examples/connection/ConnectionOptions.cs
+         :language: csharp
+         :copyable: true
+         :dedent: 8
+         :start-after: // start-builder-max-connection-life-time
+         :end-before: // end-builder-max-connection-life-time
+
+      MaxConnectionPoolSize
+      ~~~~~~~~~~~~~~~~~~~~~
+
+      .. include:: /includes/fundamentals/connection-options/maxConnectionPoolSize.rst
+
+      .. literalinclude:: /includes/fundamentals/code-examples/connection/ConnectionOptions.cs
+         :language: csharp
+         :copyable: true
+         :dedent: 8
+         :start-after: // start-builder-max-connection-pool-size
+         :end-before: // end-builder-max-connection-pool-size
+
+      MinConnectionPoolSize
+      ~~~~~~~~~~~~~~~~~~~~~
+
+      .. include:: /includes/fundamentals/connection-options/minConnectionPoolSize.rst
+
+      .. literalinclude:: /includes/fundamentals/code-examples/connection/ConnectionOptions.cs
+         :language: csharp
+         :copyable: true
+         :dedent: 8
+         :start-after: // start-builder-min-connection-pool-size
+         :end-before: // end-builder-min-connection-pool-size
+
+      WaitQueueTimeout
+      ~~~~~~~~~~~~~~~~
+
+      .. include:: /includes/fundamentals/connection-options/waitQueueTimeout.rst
+
+      .. literalinclude:: /includes/fundamentals/code-examples/connection/ConnectionOptions.cs
+         :language: csharp
+         :copyable: true
+         :dedent: 8
+         :start-after: // start-builder-wait-queue-timeout
+         :end-before: // end-builder-wait-queue-timeout
+
+Write Concern Options
+---------------------
+
+.. tabs::
+
+   .. tab:: MongoClientSettings
+      :tabid: mongo-client-settings
+
+      WriteConcern
+      ~~~~~~~~~~~~
+
+      The default write-concern settings, including write timeout and
```

Comment:
S: I know it could be correct either way, but hyphenating write-concern here is contrasting the title of the section

Reply:
i'm going to leave as is because it's a compound adjective, but i see your point